Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -3590,6 +3590,21 @@ final class HTTPClientTests: XCTestCaseHTTPClientTestsBaseClass {
         XCTAssertEqual(.ok, response.status)
     }
 
+    func testClientWithMultipath() throws {
+        do{
+            var conf = HTTPClient.Configuration()
+            conf.enableMultipath = true
+            let client = HTTPClient(configuration: conf)
+            defer {
+                XCTAssertNoThrow(try client.shutdown().wait())
+            }
+            let response = try client.get(url: self.defaultHTTPBinURLPrefix + "get").wait()
+            XCTAssertEqual(.ok, response.status)
+        }catch{
```

Comment:
We need to catch the specific error, as other errors should still fail the test. This should be [`IOError`](https://swiftpackageindex.com/apple/swift-nio/2.72.0/documentation/niocore/ioerror), with `errnoCode == EINVAL`. Please also `throw XCTSkip()` in the catch as well.